main/acle.md

```diff
@@ -9456,6 +9461,709 @@ are named after. All of the functions have external linkage.
+
+   svfloat32_t svbfmlslt[_f32](svfloat32_t zda, svbfloat16_t zn, svbfloat16_t zm);
+   svfloat32_t svbfmlslt_lane[_f32](svfloat32_t zda, svbfloat16_t zn,
+                                    svbfloat16_t zm, uint64_t imm_idx);
+   ```
+
+#### PEXT
+
+Transform a predicate-as-counter to a predicate (pair).
+
+ ``` c
+   // Variants are also available for _c16, _c32 and _c64 [SVE2.1 or SME2]
+   svbool_t svpext_lane_c8(svcount_t pnn, uint64_t imm);
+   svboolx2_t svpext_lane_c8_x2(svcount_t pnn, uint64_t imm);
+
+   ```
+
+#### PMOV
+
+ ``` c
+   // Variants are available for:
+   // _s8, _u16, _s16, _s32, _u32, _s64, _u64
+   svbool_t svpmov_lane[_u8](svuint8_t zn, uint64_t imm);
+
+   // Variants are available for:
+   // _s8, _s16, _u16, _s32, _u32, _s64, _u64
+   svbool_t svpmov[_u8](svuint8_t zn);
+
+   // Variants are available for:
+   // _s16, _s32, _u32, _s64, _u64
+   svuint16_t svpmov_lane[_u16]_m(svuint16_t zd, svbool_t pn, uint64_t imm);
+
+   // Variants are also available for:
+   // _s8, _u16, _s16, _u32, _s32, _u64, _s64
+   svuint8_t svpmov_u8_z(svbool_t pn);
+   ```
+
+#### PSEL
+
+Predicate select between predicate register or all-false.
+
+ ```c
+   // Variants are also available for _b16, _b32 and _b64 [SVE2.1 or SME]
+   svbool_t svpsel_lane_b8(svbool_t pn, svbool_t pm, uint32_t idx);
+
+   // Variants are also available for _c16, _c32 and _c64 [SVE2.1 or SME2]
+   svcount_t svpsel_lane_c8(svcount_t pn, svbool_t pm, uint32_t idx);
+   ```
+
+#### PTRUE, PFALSE
+
+Initialise predicate-as-counter to all active or all inactive.
+
+ ``` c
+   // Variants are also available for _c16, _c32 and _c64 [SVE2.1 or SME2]
+   svcount_t svptrue_c8();
+   svcount_t svpfalse_c(void);
+   ```
+
+
+#### REVD
+
+Reverse doublewords in elements.
+
+ ``` c
+   // All the intrinsics below are [SVE2.1 or SME]
+   // Variants are available for:
+   // _s8, _s16, _u16, _s32, _u32, _s64, _u64
+   // _bf16, _f16, _f32, _f64
+   svuint8_t svrevd[_u8]_m(svuint8_t zd, svbool_t pg, svuint8_t zn);
+   svuint8_t svrevd[_u8]_z(svbool_t pg, svuint8_t zn);
+   svuint8_t svrevd[_u8]_x(svbool_t pg, svuint8_t zn);
+   ```
+#### SQCVTN, SQCVTUN, UQCVTN
+
+Multi-vector saturating extract narrow and interleave.
+
+ ``` c
+   // All the intrinsics below are [SVE2.1 or SME2]
+   // Variants are also available for _u16[_s32]_x2 and _u16[_u32]_x2
+   svint16_t svqcvtn_s16[_s32]_x2(svint32x2_t zn);
+   ```
+
+#### SQRSHRN, SQRSHRUN, UQRSHRN
+
+Multi-vector saturating rounding shift right narrow and interleave.
+
+``` c
+   // All the intrinsics below are: [SVE2.1 or SME2]
+   // Variants are also available for _u16[_s32]_x2 and _u16[_u32]_x2
+   svint16_t svqrshrn_s16[_s32]_x2(svint32x2_t zn, uint64_t imm);
+   ```
+
+#### ST1D, ST1W
+
+Contiguous store of single vector operand. It is truncating store from quadword.
+
+ ``` c
+   // Variants are also available for:
+   // _u32, _s32
+   void svst1uwq[_f32](svbool_t, const float32_t *ptr, svfloat32_t data);
```

Comment:
Sorry for the very late comment, but I think we should drop the `u` from these names.  In the loads it stands for “unsigned”, but the signedness doesn't matter here, since we're simply truncating the quadwords to 32 bits.

Same for `svst1udq`.